src/components/App.js

```diff
@@ -41,8 +41,8 @@ const CurrentUserInfo = props => {
 )
 export default class App extends React.Component {
   static propTypes = {
-    userInfo: PropTypes.instanceOf(Map),
-    accessToken: PropTypes.string,
+    userInfo: PropTypes.object,
+    accessToken: PropTypes.object,
```

Comment:
Couldn't get these to work as Immutable Map instances so I chickened out and used `object`.